tests/models/test_checkpoint.py

```diff
@@ -405,6 +408,7 @@ def reference_distributed_shard(get_convert_path) -> torch.Tensor | None:
 
 
 @requires_cuda
+# NOTE: Should it depend on test_model_distributed instead?
```

Comment:
No, we can't have dependencies between `test_model` and `test_checkpoint` because we want them to run in separate processes.

Reply:
I think it needs to depend on `test_save_and_load_in_parallel` though